pkg/prototype/rice-box.go

```diff
@@ -25,34 +10,34 @@ func init() {
 	// define files
 	file2 := &embedded.EmbeddedFile{
 		Filename:    "config-map.jsonnet",
-		FileModTime: time.Unix(1526913293, 0),
+		FileModTime: time.Unix(1526918148, 0),
```

Comment:
Did these changes sneak in?

Reply:
I ran generate and it updated the mtime. I didn't see it as a problem